library/swift/src/StreamEmitter.swift

```diff
@@ -26,10 +26,11 @@ public protocol StreamEmitter: CancelableStream {
   @discardableResult
   func sendMetadata(_ metadata: [String: [String]]) -> StreamEmitter
 
-  /// End the stream after sending any provided trailers.
+  /// End the stream.
   ///
-  /// - parameter trailers: Trailers to send over the stream.
-  func close(trailers: [String: [String]])
+  /// - parameter trailers: optional trailers to send over the stream.
+  ///                       Otherwise, the stream is closed with an empty data frame.
+  func close(trailers: [String: [String]]?)
 }
 
 extension StreamEmitter {
```

Comment:
Maybe we should remove this now since it's ambiguous as to whether the stream will close with trailers or data?

Reply:
If we do this then we will be deleting any ability to send trailers from the platform layer.

Reply:
I was referring to the convenience method for closing without specifying trailers, which it looks like you already removed 😃